### ISSUE TITLE
Getting an error on the pygame example.

### ISSUE THREAD

**Issue by zeopticz** [pyimgui/pyimgui#254]
When I run the pygame integration example at [pygame.py](https://github.com/pyimgui/pyimgui/blob/master/imgui/integrations/pygame.py), it outputs this: `File "imgui\core.pyx", line 1867, in imgui.core.new_frame
ound value (need to be 0..512, or -1 for unmapped key)") at imgui-cpp/imgui.cpp:3097`
How can I fix this?

**Comment by KinoxKlark** [comment]
Yes, it is a known bug (#236) that has been fixed for the next release. You can install pyimgui from the `dev/version-2.0` branch which should be stable enough and propose a lot more functionalities. Unfortunately, I do not have as much time as I want and I won't be able to publish an official release soon.

If you absolutely want to use pyimgui 1.4 (current release) and want to fix it yourself you can take inspiration from https://github.com/pyimgui/pyimgui/blob/dev/version-2.0/imgui/integrations/pygame.py. The problem is that imgui, the cpp lib under pyimgui, only supports keycodes from -1 to 512 and pygame returns keycodes outside this range. To fix this we used a custom mapping that remap from pygame keycodes to imgui keycodes.

You should be able to fix it directly in your python environment where the lib is installed. But you can also simply create a custom implementation inspired by the proposed one in your project by copying the code in `/imgui/integrations/pygame.py` and fixing imports.

Hope this helps!
